src/Libraries/DSOfficeUtilities/DSOfficeUtilities.csproj

```diff
@@ -0,0 +1,77 @@
+    <RootNamespace>DSOffice</RootNamespace>
+    <AssemblyName>DSOfficeUtilities</AssemblyName>
+  </PropertyGroup>
+  <PropertyGroup>
+    <NoWarn>MSB3539;CS1591;NUnit2005;NUnit2007;CS0618;CS0612;CS0672</NoWarn>
+  </PropertyGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
+    <PackageReference Include="Microsoft.Office.Interop.Excel" Version="15.0.4795.1001">
+        <ExcludeAssets>all</ExcludeAssets>
+        <GeneratePathProperty>true</GeneratePathProperty>
+    </PackageReference>
+    <Reference Include="$(PkgMicrosoft_Office_Interop_Excel)\lib\netstandard2.0\Microsoft.Office.Interop.Excel.dll">
+        <EmbedInteropTypes>True</EmbedInteropTypes>
+        <WrapperTool>tlbimp</WrapperTool>
+        <Private>False</Private>
+    </Reference>
+    <COMReference Include="Microsoft.Office.Core" Condition=" '$(RuntimeIdentifier)' == 'win10-x64' ">
+        <Guid>{2DF8D04C-5BFA-101B-BDE5-00AA0044DE52}</Guid>
+        <VersionMajor>2</VersionMajor>
+        <VersionMinor>8</VersionMinor>
+        <Lcid>0</Lcid>
+        <WrapperTool>primary</WrapperTool>
+        <Isolated>False</Isolated>
+        <EmbedInteropTypes>True</EmbedInteropTypes>
+    </COMReference>
+  </ItemGroup>
+  <ItemGroup Label="Common deps">
+	<PackageReference Include="DocumentFormat.OpenXml" Version="2.12.3" CopyXML="true" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\DynamoCore\DynamoCore.csproj">
+      <Project>{7858fa8c-475f-4b8e-b468-1f8200778cf8}</Project>
+      <Name>DynamoCore</Name>
+      <Private>False</Private>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Engine\ProtoCore\ProtoCore.csproj">
+      <Project>{7a9e0314-966f-4584-baa3-7339cbb849d1}</Project>
+      <Name>ProtoCore</Name>
+      <Private>False</Private>
+    </ProjectReference>
+    <ProjectReference Include="..\..\NodeServices\DynamoServices.csproj">
+      <Project>{ef879a10-041d-4c68-83e7-3192685f1bae}</Project>
+      <Name>DynamoServices</Name>
+      <Private>False</Private>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Update="Properties\Resources.Designer.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>Resources.resx</DependentUpon>
+    </Compile>
+    <Compile Update="Properties\Resources.en-US.Designer.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>Resources.en-US.resx</DependentUpon>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Update="Properties\Resources.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+    <EmbeddedResource Update="Properties\Resources.en-US.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>Resources.en-US.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+  </ItemGroup>
+</Project>
```

Comment:
This project has all the internal APIs of the original DSOffice project and the WorkSheet and WorkBook public types.